collector/analyzer/network/protocol/dubbo/dubbo_serialize.go

```diff
@@ -0,0 +1,212 @@
+package dubbo
+
+const (
+	JSON_NEXTLINE = byte(0x0a)
+	JSON_QUTOES   = byte(0x22)
+	JSON_COLON    = byte(0x3a)
+
+	SERIAL_HESSIAN2 = byte(0x02)
+	SERIAL_FASTJSON = byte(0x06)
+)
+
+type dubboSerializer interface {
+	eatString(data []byte, offset int) int
+
+	getStringValue(data []byte, offset int) (int, string)
+
+	getStringValueByKey(data []byte, offset int, key string) string
+}
+
+var (
+	serial_hessian2  = &dubboHessian{}
+	serial_fastjson  = &dubboFastJson{}
+	serial_unsupport = &dubboUnsupport{}
```

Comment:
We are always using camel case instead of snake case.

Reply:
done